scripts/webflow-codegen.ts

```diff
@@ -0,0 +1,98 @@
+import fs from "fs"
+import path from "path"
+import { exec } from "child_process"
+import {
+  Field,
+  GetCollectionResponse,
+  ListCollectionsResponse,
+} from "../app/services/api/webflow-api.schema"
+import { Webflow } from "../app/services/api/webflow-api.service"
+import { SITE_ID } from "../app/services/api/webflow-consts"
+import { axiosInstance } from "../app/services/api/axios"
+
+const WEBFLOW_API_TOKEN = process.env.WEBFLOW_API_TOKEN
+if (!WEBFLOW_API_TOKEN) {
+  console.error("WEBFLOW_API_TOKEN is required to be passed as an environment variable.")
+  process.exit(0)
+}
+
+axiosInstance.defaults.baseURL = `https://api.webflow.com`
+axiosInstance.defaults.headers.common.Authorization = `Bearer ${WEBFLOW_API_TOKEN}`
```

Comment:
We need to hit the real webflow API instead of the proxy to access the non-collection endpoints